[2024-09-11T04:01:01.653+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T04:01:01.688+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v04.get_name scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2024-09-11T04:01:01.706+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v04.get_name scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2024-09-11T04:01:01.708+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-11T04:01:01.860+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2022-03-31 00:00:00+00:00
[2024-09-11T04:01:01.882+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=19612) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T04:01:01.886+0000] {standard_task_runner.py:72} INFO - Started process 19632 to run task
[2024-09-11T04:01:01.890+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v04', 'get_name', 'scheduled__2022-03-31T00:00:00+00:00', '--job-id', '15603', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmp5hglqezj']
[2024-09-11T04:01:01.895+0000] {standard_task_runner.py:105} INFO - Job 15603: Subtask get_name
[2024-09-11T04:01:02.033+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v04.get_name scheduled__2022-03-31T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-11T04:01:02.246+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v04' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2022-03-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-31T00:00:00+00:00'
[2024-09-11T04:01:02.251+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-11T04:01:02.259+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'fauzi'}
[2024-09-11T04:01:02.370+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-11T04:01:02.371+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v04, task_id=get_name, run_id=scheduled__2022-03-31T00:00:00+00:00, execution_date=20220331T000000, start_date=20240911T040101, end_date=20240911T040102
[2024-09-11T04:01:02.427+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-11T04:01:02.506+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-11T04:01:02.532+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
